tests/test_pinnable_slice.rs

```diff
@@ -0,0 +1,24 @@
+extern crate rocksdb;
+
+use rocksdb::{Options, DB};
+
+#[test]
+fn test_pinnable_slice() {
+    let path = "_rust_rocksdb_pinnable_slice_test";
```

Comment:
@xxuejie use `utils::DBPath` struct here please. It helps to destroy database and delete unnecessary folders with data for this test.

Reply:
Thanks for the tip! This is resolved now